servlet-feature-pack/src/main/resources/modules/system/layers/base/javax/enterprise/api/main/module.xml

```diff
@@ -34,6 +34,16 @@
     </dependencies>
 
     <resources>
-        <artifact name="${javax.enterprise:cdi-api}"/>
+        <artifact name="javax.enterprise:cdi-api:1.2">
+            <conditions>
+                <property-not-equal name="ee8.preview.mode" value="true" />
+            </conditions>
+        </artifact>
+        <artifact name="javax.enterprise:cdi-api:2.0">
```

Comment:
I think the most recent version should still be managed by the pom.xml, and only hard code the legacy version, which will hopefully be removed soonish anyway.

Reply:
Hmm, current pom is set to CDI 1.2.
We could swap it around to 2.0 and then do `<artifact name="${javax.enterprise:cdi-api}">`
Is that what you mean?

Reply:
I originally thought we should use versions from pom.xml for both APIs but Stuart's proposal to only hard code version for legacy API is reasonable.

> We could swap it around to 2.0 and then do `<artifact name="${javax.enterprise:cdi-api}">`

@manovotn +1